editor/index.html

```diff
@@ -70,6 +72,7 @@
 
 			window.editor = editor; // Expose editor to Console
 			window.THREE = THREE; // Expose THREE to APP Scripts and Console
+			window.VRButton = VRButton; // Expose VRButton to APP Scripts
```

Comment:
How do APP scripts use the VRButton?

Reply:
`app.js` from the `libs` folder appends the VRButton to the dom when the app is started, and removes it when stopped. This exposes the VRButton object to the app.js script as a global so that it can call `VRButton.create` to instantiate the button.

Reply:
Normally I'd go about this differently, but this seemed like the most simple approach since the `app.js` script is used both by the editor and published apps.